packages/test-interface-criteria/src/criteria/TestOpportunityBookable.js

```diff
@@ -1,10 +1,14 @@
 const { complement } = require('ramda');
-const { createCriteria } = require('./criteriaUtils');
+const { createCriteria, mustBeWithinBookingWindow } = require('./criteriaUtils');
 const { InternalTestOpportunityBookable } = require('./internal/InternalTestOpportunityBookable');
 const { supportsMinimalProposalFlow } = require('./sharedConstraints');
 
 const doesNotSupportMinimalProposalFlow = complement(supportsMinimalProposalFlow);
 
+function mustIfThereIsABookableWindowBeWithinIt(offer, opportunity) {
```

Comment:
This file no longer needs to be changed, now that the offerConstraint was added to InternalTestOpportunityBookable